highlight/src/lib.rs

```diff
@@ -683,6 +685,10 @@ where
             let (mut match_, capture_index) = layer.captures.next().unwrap();
             let mut capture = match_.captures[capture_index];
 
+            if layer.ignore_match_ids.contains(&match_.id()) {
```

Comment:
Since we're just doing contains lookups, a hashset would be better, no? Or actually I guess you could use just a u32 because match IDs are incrementing integers...